depthai_nodes/message/creators/map.py

```
@@ -1,9 +1,10 @@
+import depthai as dai
 import numpy as np
 
-from depthai_nodes import Map2D
 
-
-def create_map_message(map: np.ndarray, min_max_scaling: bool = False) -> Map2D:
+def create_map_message(
+    map: np.ndarray, min_max_scaling: bool = False
```

Ruff (0.16.1)

[error] Function argument `map` is shadowing a Python builtin

(A002)